### PR TITLE
Various small tweaks.

### PR DESCRIPTION
**Pull request by ejj**
These things came up as I was hacking on scale.  More to come.

### CODE REVIEW

File: cloud/credentials.go [hunk 1]
```diff
@@ -41,26 +41,25 @@ func syncCredentialsOnce(conn db.Conn, sshKey ssh.Signer, ca rsa.KeyPair) {
...
+		for _, dbm := range view.SelectFromMachine(nil) {
+			if cert, ok := ipCertMap[dbm.PublicIP]; ok {
+				dbm.PublicKey = cert
+				view.Commit(dbm)
 			}
-			return nil
-		})
-	}
+		}
+		return nil
```
Comment (kayousterhout) [inline cloud/credentials.go]: do you need this return?

Comment (ejj) [reply cloud/credentials.go]: I think so, the transaction function returns an error.  I can try it without it and see if it builds.

Comment (ejj) [reply cloud/credentials.go]: Nope doesn't build without it.

Comment (kayousterhout) [reply cloud/credentials.go]: Ahh got it didn't realize that

**Pull request merged**